docs/eventing/sources/ping-source/README.md

```diff
@@ -191,7 +191,7 @@ during the verification step in this procedure.
                 - `<cron-schedule>` is a cron expression for the schedule for the PingSource to send events, for example, `*/1 * * * *` sends an event every minute.
                 - `<content-type>` is the media type of the data you want to send, for example, `application/json`.
                 - `<base64-data>` is the base64 encoded binary data that you want to send, for example, `ZGF0YQ==`.
-                - `<sink-kind>` is any supported [Addressable](../../../reference/concepts/duck-typing.md#addressable) object that you want to use as a sink, for example, `Service` or `Deployment`.
+                - `<sink-kind>` is any supported [Addressable](../../../reference/concepts/duck-typing.md#addressable) object that you want to use as a sink, for example, `Service`.
```

Comment:
```suggestion
                - `<sink-kind>` is any supported [Addressable](../../../reference/concepts/duck-typing.md#addressable) object that you want to use as a sink, for example, a Knative Service.
```
See the style guide here for future contributions https://github.com/knative/docs/blob/main/contribute-to-docs/style-guide/word-and-phrase-list.md
Provides info about usage of terms

Reply:
No, this is referring to a Kubernetes Service

Reply:
Ahhh, I'd actually make that clear then 😉

Reply:
yeah, too many "service" concepts :)